main.rb

```diff
@@ -4,42 +4,48 @@
 require './book'
 require './rental'
 
-class Main
-  def initialize
-    @people = []
-    @books = []
-    @rentals = []
-  end
+class List
```

Comment:
- [ ] The sole purpose of the `main.rb` file should be to start your app, and that is not the case at the moment. Now, let's clean it up a bit. Kindly extract this **List** class into a different file in keeping with the Single-responsibility principle.